geopyspark/tests/python_test_utils.py

```diff
@@ -1,6 +1,6 @@
 import os
+import shutil
```

Comment:
Sorry for nitpicking, but could you please switch `import shutil` with `from os import path` so that the former is on line 3 and the latter on line 2? The python coding standard wants `imports` of the same module to be grouped together.